tests/daemon/test-circular-file.c

```diff
@@ -103,8 +103,10 @@ make_circular_file (Fixture *fixture,
                     guint64  max_size)
 {
   GError *error = NULL;
+  gboolean reinitialize = FALSE;
```

Comment:
It looks like you can't avoid this declaration and pass `FALSE` directly to `_new() `